### PR TITLE
backend/accountSummary: exclude accounts with errors

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: backend/coins/btc/account.go [hunk 1]
```diff
@@ -434,6 +436,9 @@ outer:
 
 // Balance implements the interface.
 func (account *Account) Balance() *accounts.Balance {
+	if account.fatalError {
+		return accounts.NewBalance(coin.NewAmountFromInt64(0), coin.NewAmountFromInt64(0))
```
Comment (KasparEtter) [inline backend/coins/btc/account.go]: I'm not a fan of "lying" to callers. I would prefer "proper" design by contract.

File: backend/coins/btc/account.go [hunk 2]
```diff
@@ -549,6 +554,9 @@ func (account *Account) subscribeAddress(
 
 // Transactions wraps transaction.Transactions.Transactions()
 func (account *Account) Transactions() []accounts.Transaction {
+	if account.fatalError {
+		return []accounts.Transaction{}
```
Comment (KasparEtter) [inline backend/coins/btc/account.go]: Dito. Do we call/need this? Or could we just panic (as part of the contract)?

File: backend/coins/btc/account.go (hunk 1 above)
Comment (benma) [reply backend/coins/btc/account.go]: done

File: backend/coins/btc/account.go (hunk 2 above)
Comment (benma) [reply backend/coins/btc/account.go]: done

**Pull request merged**